setup.py

```diff
@@ -510,9 +514,9 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
     'elasticsearch': elasticsearch,
     'exasol': exasol,
     'facebook': facebook,
-    'gcp': gcp,
-    'gcp_api': gcp,  # TODO: remove this in Airflow 2.1
+    'gcp_api': google,  # TODO: remove this in Airflow 2.1
     'github_enterprise': flask_oauth,
+    'google': google,
```

Comment:
Can you update "Prerequisite Tasks" section in guides?
https://airflow.readthedocs.io/en/latest/howto/operator/gcp/compute.html

Reply:
Yep!